tests/test_core.py

```diff
@@ -1866,36 +1866,36 @@ def test_life(self):
     src = open(path_from_root('tests', 'life.c'), 'r').read()
     self.do_run(src, '''--------------------------------
 []                                    []                  [][][]
-                    []  []    []    [][]  []            []  []  
```

Comment:
this whitespace actually matters, I'm afraid - the test expects it the way it was before your change